doc/quickstart/geostyler_quickstart.rst

```diff
@@ -0,0 +1,139 @@
+
+Import the GeoStyler ``Style`` component into your application as follows:
+
+.. code-block:: javascript
+
+    import { Style } from 'geostyler';
+
+The ``Style`` component provides a variety of styling functionality.
+
+After importing, you can use it in your render method like this:
+
+.. code-block:: html
+
+    <Style
+        compact={true}
+    />
+
+The ``Style`` component should now be visible in the browser.
+
+.. image:: /images/projects/geostyler/quickstart_compact.png
+
+Integrate geostyler-sld-parser
+==============================
+
+To edit an existing SLD via GeoStyler, we first have to import and instantiate the ``geostyler-sld-parser``.
+
+.. code-block:: javascript
+
+    import SLDParser from 'geostyler-sld-parser';
+    const parser = new SLDParser();
+
+Afterwards, we are able to read an existing SLD into the GeoStyler-readable format.
+
+.. code-block:: javascript
+
+    parser.readStyle('<?xml version="1.0" encoding="UTF-8"?><sld:StyledLayerDescriptor xmlns="http://www.opengis.net/sld" xmlns:sld="http://www.opengis.net/sld" xmlns:gml="http://www.opengis.net/gml" xmlns:ogc="http://www.opengis.net/ogc" version="1.0.0"> <sld:NamedLayer> <sld:Name>Default Styler</sld:Name> <sld:UserStyle> <sld:Name>Default Styler</sld:Name> <sld:Title>Gravel_Program_2016</sld:Title> <sld:FeatureTypeStyle> <sld:Name>name</sld:Name> <sld:Rule> <sld:MinScaleDenominator>1.0</sld:MinScaleDenominator> <sld:MaxScaleDenominator>1.0E7</sld:MaxScaleDenominator> <sld:LineSymbolizer> <sld:Stroke> <sld:CssParameter name="stroke">#8000FF</sld:CssParameter> <sld:CssParameter name="stroke-width">3.000</sld:CssParameter> </sld:Stroke> </sld:LineSymbolizer> </sld:Rule> </sld:FeatureTypeStyle> </sld:UserStyle> </sld:NamedLayer> </sld:StyledLayerDescriptor>')
+        .then(style => console.log(style));
+
+
+To connect this style with the UI, we have to store it in a state variable and pass it to
+our ``Style`` component. After defining ``myStyle`` as a state variable, you can use it as follows:
+
+.. code-block:: javascript
+
+    parser.readStyle('<?xml version="1.0" encoding="UTF-8"?><sld:StyledLayerDescriptor xmlns="http://www.opengis.net/sld" xmlns:sld="http://www.opengis.net/sld" xmlns:gml="http://www.opengis.net/gml" xmlns:ogc="http://www.opengis.net/ogc" version="1.0.0"> <sld:NamedLayer> <sld:Name>Default Styler</sld:Name> <sld:UserStyle> <sld:Name>Default Styler</sld:Name> <sld:Title>Gravel_Program_2016</sld:Title> <sld:FeatureTypeStyle> <sld:Name>name</sld:Name> <sld:Rule> <sld:MinScaleDenominator>1.0</sld:MinScaleDenominator> <sld:MaxScaleDenominator>1.0E7</sld:MaxScaleDenominator> <sld:LineSymbolizer> <sld:Stroke> <sld:CssParameter name="stroke">#8000FF</sld:CssParameter> <sld:CssParameter name="stroke-width">3.000</sld:CssParameter> </sld:Stroke> </sld:LineSymbolizer> </sld:Rule> </sld:FeatureTypeStyle> </sld:UserStyle> </sld:NamedLayer> </sld:StyledLayerDescriptor>')
+        .then(style => this.setStyle({myStyle}));
+
+    // ...
```

Comment:
Sphinx is complaining about code not been js
```
geostyler_quickstart.rst:98: WARNING: Could not lex literal_block as "js". Highlighting skipped.
```

So, better remove the `js` on this code-block